<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="contentWidget" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>115</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionOptionsSettings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSqlScript"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-exit.png</normaloff>:/images/file-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/help-about.png</normaloff>:/images/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSqlScript">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/script.png</normaloff>:/icons/script.png</iconset>
   </property>
   <property name="text">
    <string>SQL</string>
   </property>
  </action>
  <action name="actionOptionsSettings">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/options-settings.png</normaloff>:/images/options-settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
